packages/contracts/contracts/discovery/Controller.sol

```diff
@@ -31,6 +38,29 @@ contract Controller is IController {
         revert("not implemented");
     }
 
+    function registerProject(
+        address _token,
+        uint256 _saleSupply,
+        uint256 _rate
+    ) external {
```

Comment:
maybe add this as a future task but: whoever calls this needs to have a KYC done on the fractal registry, and included in a specific list (whose ID we don't have yet)

Reply:
added as a issue 👍